pkg/config/model.go

```diff
@@ -261,15 +233,10 @@ func ApplyJSON(s Spec, js string) error {
 		}
 	}
 
-	// gogo protobuf
-	if pb, ok := s.(gogoproto.Message); ok {
-		err := gogoprotomarshal.ApplyJSON(js, pb)
-		return err
-	}
-
 	return json.Unmarshal([]byte(js), &s)
 }
 
+// DeepCopy the object. Note: this does not use the correct DeepCopy on Kubernetes types; only use with Istio types.
 func DeepCopy(s any) any {
```

Comment:
Did you check out existing DeepCopy usage?

Reply:
yep, its all in the description of https://github.com/istio/istio/pull/57883